portfolio/src/main/webapp/style.css

```diff
@@ -10,6 +10,10 @@
     margin-left: 350px;
 }
 
+.logo {
+    margin-left: 1200px;
```

Comment:
doing margins with pixels this large can be tricky. What if the screen size changes dramatically? I like to use percentages for this reason in this context. I suggest playing around with screen size (shrink the size of the window to emulate a smaller screen, or shrink the size of the web page to emulate a larger screen) to get a feel of how this would look at different sizes.